polymer/slambook/lib/tute_slambookform.dart

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+@HtmlImport('tute_slambookform.html')
+library slambook.lib.tute_slambookform;
+
+import 'dart:html';
+import 'dart:convert';
+import 'package:polymer/polymer.dart';
+import 'package:web_components/web_components.dart' show HtmlImport;
+
+@PolymerRegister('tute-slambookform', extendsTag: 'form')
+class SlamBookComponent extends FormElement with 
+    PolymerMixin, PolymerBase, JsProxy {
+
+  AllAboutMe get myData => _myData;
+  AllAboutMe _myData = new AllAboutMe();
+
+  @Property(observer: 'submitForm')
```

Comment:
This should probably be on the getter above instead of the setter, I am actually surprised this works :)